schema/dataKeys.json

```diff
@@ -0,0 +1,352 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "fbc": {
+      "type": "object",
+      "properties": {
+        "request": {
+          "type": "string",
+          "description": "The internal pipline name to handle requests e.g. iib"
+        },
+        "fbcPublishingCredentials": {
+          "type": "string",
+          "description": ""
+        },
+        "buildTags": {
+          "type": "array",
+          "description": "",
+          "items": {
+            "type": "string"
+          }
+        },
+        "addArches": {
+          "type": "array",
+          "description": "",
+          "items": {
+            "type": "string"
+          }
+        },
+        "hotfix": {
+          "type": "boolean",
+          "description": "Indicates if the build is a hotfix"
+        },
+        "preGA": {
+          "type": "boolean",
+          "description": "Indicates if the build is a preGA"
+        },
+        "stagedIndex": {
+          "type": "boolean",
+          "description": "Indicates if the build is a staged index"
+        },
+        "productName": {
+          "type": "string",
+          "description": "The product name e.g. Red Hat Product "
+        },
+        "productVersion": {
+          "type": "string",
+          "description": "The product version e.g v1.0.0"
+        },
+        "buildTimeoutSeconds": {
+          "type": "integer",
+          "description": "The build timeout in seconds e.g. 1500"
+        },
+        "requestTimeoutSeconds": {
+          "type": "integer",
+          "description": "The requested timeout in seconds e.g. 1500"
+        },
+        "timestampFormat": {
```

Comment:
this field is used to define the timestamp format that will be used by `date` command  to construct the preGA tag. The default is `%s` (or seconds since 1970) but can any format allowed by `date` can be used.

Reply:
Added a description  `"description": "The timestamp format which defaults to %s e.g. %Y-%m-%d"`